src/templates/blog-post-archive.tsx

```diff
@@ -22,36 +39,54 @@ const BlogIndex = ({ data, pageContext: { nextPagePath, previousPagePath } }) =>
   return (
     <Layout>
       <SEO title="All posts" />
-      <Bio />
-      <ol>
-        {posts.map((post) => {
-          const title = post.title;
+      <Container style={{ paddingTop: "50px" }}>
+        <H1 style={{ gridColumn: "1/13" }}>Article Archive</H1>
```

Comment:
This seems to mess up the width of the archive column when in mobile mode. :( not sure about directly adding CSS Grid stuff to text elements, either